src/tests/bdd/entity/validate_default_entity.feature

```diff
@@ -0,0 +1,117 @@
+Feature: Validate Default Entity
+
+  Background: There are data sources in the system
+    Given the system data source and SIMOS core package are available
+    Given there are data sources
+      | name             |
+      | data-source-name |
+    Given there are repositories in the data sources
+      | data-source       | host | port  | username | password | tls   | name     | database  | collection | type     | dataTypes |
+      | data-source-name  | db   | 27017 | maf      | maf      | false | repo1    |  bdd-test | entities   | mongo-db | default   |
+
+    Given there exist document with id "1" in data source "data-source-name"
+    """
+    {
+      "name": "root_package",
+      "description": "",
+      "type": "dmss://system/SIMOS/Package",
+      "isRoot": true,
+      "content": [
+        {
+          "address": "$2",
+          "type": "dmss://system/SIMOS/Reference",
+          "referenceType": "storage"
+        }
+      ]
+    }
+    """
+    Given there exist document with id "2" in data source "data-source-name"
+    """
+    {
+      "name": "PersonBlueprint",
+      "type": "dmss://system/SIMOS/Blueprint",
+      "attributes": [
+        {
+          "name": "type",
+          "type": "dmss://system/SIMOS/BlueprintAttribute",
+          "attributeType": "string",
+          "optional": false
+        },
+        {
+          "name": "name",
+          "type": "dmss://system/SIMOS/BlueprintAttribute",
+          "attributeType": "string",
+          "label": "Name",
+          "optional": false
+        },
+        {
+          "name": "phoneNumber",
+          "type": "dmss://system/SIMOS/BlueprintAttribute",
+          "attributeType": "integer",
+          "label": "Phone number",
+          "optional": true,
+          "default": 123445566778
+        },
+        {
+          "name": "nationality",
+          "type": "dmss://system/SIMOS/BlueprintAttribute",
+          "attributeType": "string",
+          "label": "Nationality",
+          "optional": false,
+          "default": "Norwegian"
+        },
+        {
+          "name": "prime_minister",
+          "type": "dmss://system/SIMOS/BlueprintAttribute",
+          "attributeType": "dmss://data-source-name/root_package/PersonBlueprint",
+          "label": "Prime Minister",
+          "optional": true,
+          "default": {
+            "type": "dmss://data-source-name/root_package/PersonBlueprint",
+            "name": "Jonas",
+            "nationality": "Norwegian"
+          }
+        },
+        {
+          "name": "lngLat",
```

Comment:
Could we have an array of complex values too?

Reply:
Added a couple of optional complex parents 👪